lib/Target/SubstraitPB/Import.cpp

```diff
@@ -153,6 +184,154 @@ static mlir::FailureOr<mlir::Type> importType(MLIRContext *context,
   }
 }
 
+mlir::FailureOr<CallOp>
+importAggregateFunction(ImplicitLocOpBuilder builder,
+                        const AggregateFunction &message) {
+  MLIRContext *context = builder.getContext();
+  Location loc = UnknownLoc::get(context);
```

Comment:
Why not use the location from the ImplicitLocOpBuilder?

Reply:
(same below)

Reply:
Good idea! Fixed in latest commit.